### PR TITLE
Automated the account test case NP-4809

### PR DESCRIPTION
**Pull request by shivanand-jangam**
Commit details:
1. Added class.java
2. modified class2.java method name sakjdvjskdnvc
3. added extra wait time in common util for the method aljvcjksdnvcu

### CODE REVIEW

File: QEA-Integration-Tests/src/test/java/com/pojos/userdetails/GitHubTest.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.pojos.userdetails;
+
+public class GitHubTest {
+
+	public void testGit() {
+		System.out.println("Testing Github");
+	}
+
+	public void testGit2() {
+		System.out.println("Testing Github2");
```
Comment (VijayDYaligar) [inline QEA-Integration-Tests/src/test/java/com/pojos/userdetails/GitHubTest.java]: Can you please modify this line and update wit new line

File: QEA-Integration-Tests/src/test/java/com/pojos/userdetails/GitHubTest.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.pojos.userdetails;
+
+public class GitHubTest {
+
+	public void testGit() {
+		System.out.println("Testing Github");
```
Comment (VijayDYaligar) [inline QEA-Integration-Tests/src/test/java/com/pojos/userdetails/GitHubTest.java]: Test comment

**Review by VijayDYaligar — COMMENTED** [review]
Implement review comment

**Review by shivanand-jangam — COMMENTED** [review]
Imple,mented review changes

**Review by VijayDYaligar — APPROVED** [review]
Changes looks good

**Pull request merged**